src/HashBus.Viewer.MostMentioned/LeaderboardView.cs

```diff
@@ -121,26 +121,25 @@ static class LeaderboardView<TEntry> where TEntry : WebApi.IEntry
                 ColorConsole.WriteLine();
                 foreach (var line in lines)
                 {
-                    ColorConsole.WriteLine(new ColorToken[] { padding }.Concat(line).ToArray());
+                    ColorConsole.WriteLine(new ColorToken[]
+                    {
+                        padding
+                    }.Concat(line).ToArray());
                 }
 
                 ColorConsole.WriteLine(
                     padding,
-                    $"Total {itemsName}:".Gray(),
-                    " ",
-                    $"{currentLeaderboard?.Count ?? 0:N0}"
-                        .Color(currentLeaderboard?.Count - previousLeaderboard?.Count > 0 ? movementColors[-1] : movementColors[0]),
-                    $" since ".Gray(),
-                    $"{currentLeaderboard.Since?.ToLocalTime().ToString() ?? ("started")}".Gray());
+                    $"{currentLeaderboard.Since?.ToLocalTime().ToString() ?? ("started")} to {currentLeaderboard.LastActivityDate?.ToLocalTime().ToString() ?? ("started")}".Gray());

```

Comment:
When the datetimes are not null, it says

> 12:23 1.12.2015 to 14:34 10.12.2015

so the question is what should it say when at least one of those datetimes is null?

In the current version, when the single datetime is not null, it says

> Total tweets: 34 since 12:23 1.12.2015

and when it is null, it says

> Total tweets: 34 since started

This is why the simple text substitution works. It's not so easy with new format.